### DISCUSSION TITLE
Disable mouse click moving the cursor

### DISCUSSION THREAD

**Discussion by grimm26** [kovidgoyal/kitty#5052] [Q&A] [answered]
When I am on macos and need to click into an os window to focus it before I paste something, the click moves the cursor and I end up pasting into the wrong spot. I don't ever want the mouse to move my cursor in a terminal. How do I keep this from happening?

**Comment by kovidgoyal** [comment, marked as answer]
Read the docs
https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Click-the-link-under-the-mouse-or-move-the-cursor

**Reply by grimm26** [reply]
Excellent, thank you!  I'm new and there's a lot to read :)

**Comment by kovidgoyal** [comment]
On Wed, May 04, 2022 at 08:22:59AM -0700, Mark Keisler wrote:
> Excellent, thank you!  I'm new and there's a lot to read :)

That there is, kitty is extremely not barebones :)
